x11-wm/lumina/lumina-1.2.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+DESCRIPTION="Lumina desktop environment"
+HOMEPAGE="http://lumina-desktop.org/"
+I18N="161211"
+SRC_URI="https://github.com/trueos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~grozin/${PN}-i18n-${I18N}.tar.bz2"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE=""
+LANGS="af ar az bg bn bs ca cs cy da de el en_GB en_ZA es et eu fa fi fr fr_CA fur gl he hi hr hu id is it ja ka ko lt lv mk mn ms mt nb ne nl pa pl pt pt_BR ro ru sa sk sl sr sv sw ta tg th tr uk ur uz vi zh_CN zh_HK zh_TW zu"
+for lang in ${LANGS}; do
+	IUSE="${IUSE} linguas_${lang}"
+done
+
+COMMON_DEPEND="dev-qt/qtcore:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtmultimedia:5[widgets]
+	dev-qt/qtsvg:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	dev-qt/qtgui:5
+	dev-qt/qtdeclarative:5
+	x11-libs/libxcb:0
+	x11-libs/xcb-util
+	x11-libs/xcb-util-image
+	x11-libs/xcb-util-wm"
+
+DEPEND="$COMMON_DEPEND
+	dev-qt/linguist-tools:5"
+
+RDEPEND="$COMMON_DEPEND
+	kde-frameworks/oxygen-icons
+	x11-misc/numlockx
+	x11-wm/fluxbox
+	x11-apps/xbacklight
+	media-sound/alsa-utils
+	sys-power/acpi
+	app-admin/sysstat"
+
+src_prepare(){
+	default
+
+	rm -rf src-qt5/desktop-utils || die
+
+	sed -e "/desktop-utils/d" -i src-qt5/src-qt5.pro || die
+}
+
+src_configure(){
+	eqmake5 PREFIX="${ROOT}usr" L_BINDIR="${ROOT}usr/bin" \
```

Comment:
`${ROOT}` is not allowed and makes no sense in `src_*` functions, as it's only used for cross-compiling. You probably meant to use `${EPREFIX}` (which is not slash-terminated)